packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -179,6 +175,13 @@ async function fundPayment(
       return FundingError.WrongState
     }
     if (amount !== payment.sendAmount.value) return FundingError.InvalidAmount
+
+    // Create the outgoing payment liquidity account before trying to transfer funds to it.
+    await deps.accountingService.createLiquidityAccount({
+      id: id,
+      asset: payment.asset
+    })
```

Comment:
See https://github.com/interledger/rafiki/pull/432/files#r951469744; I think this should still check & ignore `exists` errors.